source/index.pug

```diff
@@ -19,20 +19,93 @@ html
 
     title Women and Web Literacy
   body
-    header.container
-      img(src='images/logo.png')
+    header.p-y-3
+      .container
+        a.mozilla-logo(href='https://www.mozilla.org')
+          img(src='images/mozilla-wordmark.svg',target='_blank').m-b-1
 
-  .container
-    .row
+        h1.center-text Women & Web Literacy
+        p.center-text Working towards an open, safe and innovative web for women everywhere.
+
+  .container.m-y-2
+    .row.m-b-1
       .col-sm-12
-        include:markdown markdown/index.md
+        h2 What We Do
+    .row
+      .col-md-6
+        include:markdown markdown/what-we-do/1.md
```

Comment:
can we name these two files as something a little more meaningful? e.g. `description.md` and `principles.md`